components/atoms/Search/search.tsx

```diff
@@ -83,19 +85,22 @@ const Search = ({
         onFocus={() => setShowSuggestions(true)}
         onBlur={() => setTimeout(() => setShowSuggestions(false), 500)}
       />
-
       {suggestions && suggestions.length > 0 && showSuggestions && (
         <div className="absolute left-0 z-10 w-full pb-1 space-y-1 bg-white border rounded-lg cursor-pointer shadow-input border-light-slate-6 top-full">
-          {suggestions.map((suggestion, index) => (
-            <div
-              className="px-4 py-2 overflow-hidden break-all text-light-slate-9 hover:bg-light-slate-2"
-              style={suggestionsStyle}
-              key={index}
-              onClick={() => handleOnSelect(suggestion)}
-            >
-              <span className="pl-5 text-sm ">{suggestion}</span>
-            </div>
-          ))}
+          <ScrollArea type="auto">
+            <ScrollViewport className="max-h-60 w-full">
```

Comment:
Remove the usage of `ScrollViewport`, use only the `ScrollArea` with a fixed height

```javascript
<ScrollArea type="auto" className="h-60">
  ...
</ScrollArea>
```
Using the `ScrollViewport` assumes you want extra customization to the T! but you don't have to.